MAUI/Carousel-View/Populating-Data.md

```diff
@@ -0,0 +1,275 @@
+---
+layout : post
+title: Populating Data in .NET MAUI Carousel View control | Syncfusion
+description: Learn here all about Populating Data support in Syncfusion .NET MAUI Carousel View (SfCarousel) control and more.
+platform : maui
+control : Carousel
+documentation : ug
+---
+
+# Populating Data in .NET MAUI Carousel View (SfCarousel)
+
+`SfCarousel` control, supports binding to different data sources such as IList Data Source, Observable Collection Data Source.
```

Comment:
`SfCarousel` control supports binding to different data sources such as IList Data Source and Observable Collection Data Source.

Reply:
Addressed the changes.